### PR TITLE
feat: search ui - recent chats #2249

### LINKED ISSUE #2249: Search - Recent chats

### PR DESCRIPTION
**Pull request by frolyo**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyContactedPeople.razor [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    protected override ComputedState<IReadOnlyList<Chat>>.Options GetStateOptions()
+        => ComputedStateComponent.GetStateOptions(GetType(),
+            static t => new ComputedState<IReadOnlyList<Chat>>.Options() {
+                InitialValue = [],
+                Category = ComputedStateComponent.GetStateCategory(t),
+            });
+
+    protected override async Task<IReadOnlyList<Chat>> ComputeState(CancellationToken cancellationToken) {
+        var session = Hub.Session();
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyContactedPeople.razor]: Better to extract shortcut property Session

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@{
+    var chats = State.ValueOrDefault ?? [];
+    if (chats.Count == 0)
+        return;
+}
+
+<div class="recently-viewed-groups">
+    <div class="c-header">
+        <div class="c-title">Recently viewed</div>
+        @* <HeaderButton Class="blackout"> *@
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor]: Do you still need this markup in comments?

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    protected override ComputedState<IReadOnlyList<ChatInfo>>.Options GetStateOptions()
+        => ComputedStateComponent.GetStateOptions(GetType(),
+            static t => new ComputedState<IReadOnlyList<ChatInfo>>.Options() {
+                InitialValue = [],
+                Category = ComputedStateComponent.GetStateCategory(t),
+            });
+
+    protected override async Task<IReadOnlyList<ChatInfo>> ComputeState(CancellationToken cancellationToken) {
+        var session = Hub.Session();
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor:36]: Better to extract shortcut property Session

File: src/dotnet/Users.Service/Module/UsersDbInitializer.cs [hunk 4]
```diff
@@ -51,22 +54,76 @@ private async Task EnsureTestBotsExist(CancellationToken cancellationToken)
...
-        var account = await accountsBackend.Get(userId, cancellationToken).ConfigureAwait(false);
-        return account;
+        return await accountsBackend.Get(userId, cancellationToken).ConfigureAwait(false);
     }
 
-    private async Task<AccountFull> AddInternalAccount(UserId userId, string name, CancellationToken cancellationToken)
+    private async Task<AccountFull> AddInternalAccount(InternalUserInfo userInfo, CancellationToken cancellationToken)
     {
+        var userId = userInfo.Id;
+        var username = userInfo.UserNameOrDefault;
```
Comment (crui3er) [inline src/dotnet/Users.Service/Module/UsersDbInitializer.cs]: rename to `userName`

**Review by crui3er — APPROVED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor (hunk 3 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor:36]: done

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor (hunk 2 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor]: removed

File: src/dotnet/Users.Service/Module/UsersDbInitializer.cs (hunk 4 above)
Comment (frolyo) [reply src/dotnet/Users.Service/Module/UsersDbInitializer.cs]: done

**Pull request merged**